riscv-sbi.adoc

```diff
@@ -201,6 +202,8 @@ error codes.
 |  SBI_ERR_ALREADY_AVAILABLE | -6
 |  SBI_ERR_ALREADY_STARTED   | -7
 |  SBI_ERR_ALREADY_STOPPED   | -8
+|  SBI_ERR_BUSY              | -9
+|  SBI_ERR_OVERFLOW          | -10
```

Comment:
From the PR review: A related question about the semantics (and the associated name). Since the response would overflow the passed in buffer, the presumably the passed-in input buffer was too small, so perhaps it should be InsufficientCapacity / BufferTooSmall, or something like that?